moVirt/src/main/java/org/ovirt/mobile/movirt/ui/dashboard/maps/HostStatusMap.java

```diff
@@ -0,0 +1,45 @@
+package org.ovirt.mobile.movirt.ui.dashboard.maps;
+
+import org.ovirt.mobile.movirt.model.Host;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by suomiy on 5/18/16.
+ */
+public enum HostStatusMap {
```

Comment:
also here Im not sure... The warning should say that there is something wrong there and the "non_responsive" and "non_operational" are two of the most common statuses if the host has problems and requires some action from the admin.